docs/installation.md

```diff
@@ -113,6 +113,22 @@ terminal session and verify `pipx` does run.
 > It is not recommended to install `pipx` via `pipx`. If you'd like to do this anyway, take a look at the
 > [`pipx-in-pipx`](https://github.com/mattsb42-meta/pipx-in-pipx) project and read about the limitations there.
 
+
+### On FreeBSD:
+
+- Install via package manager
+
+```sh
+pkg install -y py311-pipx
```

Comment:
Maybe users should also run `pipx ensurepath` if the package manager doesn't do this automatically?
```suggestion
pkg install -y py311-pipx
pipx ensurepath
```